wg-feature-store/charter.md

```diff
@@ -0,0 +1,42 @@
+# WG Feature Store Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Feature Store covers the definition, management, storage, discovery, and serving of features to models.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Provide a unified feature serving layer for both model training and online serving.
```

Comment:
Is the word serving correct? (Since the WG/SIG is called Feature **Store**)

Reply:
Yes, it is correct. Providing a consistent interface to accessing data in both training and online serving is one of the essential parts of a feature store.

Reply:
Happy to use a different term like "access" or "retrieval" if we want to disambiguate it.